### PR TITLE
Adds external service endpoints

### PR DESCRIPTION
**Pull request by Timunas**
This commit adds the external service httpStatus and timeout endpoints that will enable the automation of External Gateway tests.

JIRA: EDGE-1832

### CODE REVIEW

File: src/main/kotlin/com/talkdesk/automation/testing/service/AutomationTestingServiceApplication.kt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.talkdesk.automation.testing.service
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.runApplication
+import java.util.UUID
+
+@SpringBootApplication
+class AutomationTestingServiceApplication {
+    companion object {
+        val INSTANCE_ID = UUID.randomUUID().toString().replace("-", "")
```
Comment (jaime-talkdesk) [inline src/main/kotlin/com/talkdesk/automation/testing/service/AutomationTestingServiceApplication.kt]: Do we need this instance ID? We have meza hostname which has the pod name

File: src/test/kotlin/com/talkdesk/automation/testing/service/controller/ExternalServiceControllerTest.kt [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    ) {
+        mvc.perform(httpMethodRequestBuilder("$httpStatusRoute/500")).andExpect(status().isInternalServerError)
+    }
+
+    @ParameterizedTest
+    @ArgumentsSource(HttpMethodArgumentsProvider::class)
+    fun `handleHttpStatusCodeRequest - should return Bad Request with message when code isn't valid`(
+        httpMethodRequestBuilder: (String) -> MockHttpServletRequestBuilder
+    ) {
+        mvc.perform(httpMethodRequestBuilder("$httpStatusRoute/999")).andExpect(status().isBadRequest).andExpect {
```
Comment (jaime-talkdesk) [inline src/test/kotlin/com/talkdesk/automation/testing/service/controller/ExternalServiceControllerTest.kt]: Why don't you just receive the HTTP Method?

File: src/main/kotlin/com/talkdesk/automation/testing/service/AutomationTestingServiceApplication.kt (hunk 1 above)
Comment (jaime-talkdesk) [reply src/main/kotlin/com/talkdesk/automation/testing/service/AutomationTestingServiceApplication.kt]: Aren't you missing the /ping and /health endpoints?

Comment (Timunas) [reply src/main/kotlin/com/talkdesk/automation/testing/service/AutomationTestingServiceApplication.kt]: We were using the actuator and the instance ID was a leftover. I will check if we can change the port for the healthcheck controller only

**Pull request merged**